qiskit/_quantumprogram.py

```diff
@@ -344,7 +354,7 @@ def create_circuit(self, name=None, qregisters=None, cregisters=None):
                 Quantum Program
         """
         if name is None:
-            name = self._create_id('qc', self.__quantum_program.keys())
+            name = self._create_id('qc%i', self.__quantum_program.keys())
```

Comment:
Can you check if this is ok, in the same spirit as the comment for `create_classical_register`?

Reply:
Also fixed in cbc4b4e